src/Common/src/Interop/Unix/libcurl/Interop.SafeCurlHandle.cs

```diff
@@ -37,6 +40,19 @@ protected override bool ReleaseHandle()
 
         internal sealed class SafeCurlMultiHandle : SafeHandle
         {
+            private readonly CancellationTokenSource _cancellationTokenSource = new CancellationTokenSource();
```

Comment:
Oh I see.. you mean that in RemoveEasyHandle I can just use a bool. Will do this in next set of changes